pkg/model/api/v1/dashboard.go

```diff
@@ -27,7 +27,7 @@ import (
 type PanelSpec struct {
 	Display common.Display `json:"display" yaml:"display"`
 	Plugin  common.Plugin  `json:"plugin" yaml:"plugin"`
-	Queries []Query  `json:"queries" yaml:"queries"`
+	Queries []Query        `json:"queries" yaml:"queries"`
```

Comment:
since it's an optional field, it should be marked by the tag "omitempty"
```suggestion
	Queries []Query        `json:"queries,omitempty" yaml:"queries,omitempty"`
```